sv_inline.h

```diff
@@ -0,0 +1,527 @@
+/*    sv.h
+ *
+ *    Copyright (C) 1991, 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999, 2000,
+ *    2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009 by Larry Wall and others
+ *
+ *    You may distribute under the terms of either the GNU General Public
+ *    License or the Artistic License, as specified in the README file.
+ *
+ */
+
+
+/* 2022 */
+/* BLAH BLAH BLAH */
+
+/* This came from perl.h*/
+
+/* The old value was hard coded at 1008. (4096-16) seems to be a bit faster,
+   at least on FreeBSD.  YMMV, so experiment.  */
+#ifndef PERL_ARENA_SIZE
+#define PERL_ARENA_SIZE 4080
+#endif
+
+/* This came from sv.h*/
+
+#ifdef PERL_POISON
+#  define SvARENA_CHAIN(sv)     ((sv)->sv_u.svu_rv)
+#  define SvARENA_CHAIN_SET(sv,val)     (sv)->sv_u.svu_rv = MUTABLE_SV((val))
+/* Whilst I'd love to do this, it seems that things like to check on
+   unreferenced scalars
+#  define POISON_SV_HEAD(sv)    PoisonNew(sv, 1, struct STRUCT_SV)
+*/
+#  define POISON_SV_HEAD(sv)    PoisonNew(&SvANY(sv), 1, void *), \
+                                PoisonNew(&SvREFCNT(sv), 1, U32)
+#else
+#  define SvARENA_CHAIN(sv)     SvANY(sv)
+#  define SvARENA_CHAIN_SET(sv,val)     SvANY(sv) = (void *)(val)
+#  define POISON_SV_HEAD(sv)
+#endif
```

Comment:
Unless I'm missing something, none of the above defines had to be moved here.

Reply:
`SvARENA_CHAIN` is used in the `uproot_SV` macro below. 

I moved the whole block of `define`s that seemed to belong together, rather than breaking them up.